lib/client_int.h

```diff
@@ -76,7 +76,7 @@ typedef struct SF_STAGE_INFO {
   char *path;
   char *region;
   char *storageAccount; // For Azure only
-  char *endPoint; //For Azure only.
+  char *endPoint; //For FIPS support
```

Comment:
nit: mention that this is for azure and fips support (like comment in PutGetParseResponse.hpp)